### DISCUSSION TITLE
Better scroll with track pad

### DISCUSSION THREAD

**Discussion by GIFO14** [isontheline/pro.webssh.net#968] [Ideas]
I am a Logitech Combo Touch (iPad Air 5th) user and I cannot scroll correctly using the track pad while I’m in a file. Maybe it’s just a Combo Touch issue, but if it is not, I would greatly appreciate being able to scroll as usual. I’m using my iPad as a desktop computer too, and having this issue prevents me from using the editor in my external monitor (since I need to scroll using my fingers).

**Comment by isontheline** [comment]
Thank you so much for your feedback @GIFO14 🙏

Are we talking about the text editor inside the SFTP connection?

**Reply by GIFO14** [reply]
Yes, exactly.

**Reply by isontheline** [reply]
Thank you!

I will so open an issue in order to track your request and enhance WebSSH scrolling within the SFTP text editor : #969

I can't you any ETA but I'm planning it to upcoming milestones.
